datasets/huishoudelijkafval/huishoudelijkafval.json

```diff
@@ -185,7 +185,75 @@
             "relation": "bag:nummeraanduidingen",
             "uri": "https://bag.basisregistraties.overheid.nl/def/bag#identificatiecode",
             "description": "Identificatie nummeraanduiding"
-          }
+          },
+          "containerRalKleurNaam": {
+            "provenance": "container_ral_kleur_naam",
+            "type": "string",
+            "description": "De naam van de ralkelur"
+          },
+          "containerRalKleurCode": {
+            "provenance": "container_ral_kleur_code",
+            "type": "string",
+            "description": "De internationale ralkleurcode."
+          },
+          "containerRalKleurHexcode": {
+            "provenance": "container_ral_kleur_hexcode",
+            "type": "string",
+            "description": "De hexcode van de ralkleur."
+          },
+          "containerChipNummber": {
+            "provenance": "container_chip_nummber",
+            "type": "string",
+            "description": "Het identificatienummer van de chip die registreert wanneer de container wordt geleegd."
+          },
+          "containerUnitCardLezerId": {
+            "provenance": "container_unit_card_lezer_id",
+            "type": "string",
+            "description": "Het identificatienummer van de kaartlezer."
+          },
+          "containerKleur": {
+            "provenance": "container_kleur",
+            "type": "string",
+            "description": "De naam van de kleur"
+          },
+          "containerMark": {
+            "provenance": "container_mark",
+            "type": "integer",
+            "description": ""
+          },
+          "containerDatumVervanging": {
+            "provenance": "container_datum_vervanging",
+            "type": "string",
+            "format": "date",
+            "description": "De datum waarop de container wordt vervangen."
+          },
+          "containerDatumWijziging": {
+            "provenance": "container_datum_wijziging",
+            "type": "string",
+            "format": "date-time",
+            "description": "Datum waarop de container is gewijzigd."
+          },
+          "containerEndOfLife": {
+            "provenance": "container_end_of_life",
+            "type": "string",
+            "format": "date",
+            "description": "Datum waarop het object het einde van zijn levenscyclus heeft bereikt."
+          },
+          "containerEigenaarschap": {
+            "provenance": "container_eigenaarschap",
+            "type": "string",
+            "description": "De soort eigenaarschap van het oject."
+          },
+          "containerEigenaarschapOpmerking": {
+            "provenance": "container_eigenaarschap_opmerking",
+            "type": "string",
+            "description": "Beschrijving van  het type de eigenaarschap."
```

Comment:
Typo: `type van de ..`. Ik twijfel nog of het nu `de` of `het` eigenaarschap is (ik neig naar `het`).

Reply:
Volgens mij die de kan weg. Volgens [https://www.welklidwoord.nl/eigenaarschap](https://www.welklidwoord.nl/eigenaarschap) is "de" eigenaarschap :-)